doc/dev/api/server/Ppx_syntax.wiki

```diff
@@ -0,0 +1,49 @@
+=RPC ppx syntax extension
+
+The ocsigen-start syntax extension allows to write, in one function, both the
+client and server part of your RPC.
+
+
+==Constructions
+
+Three types of constructions are available to build an RPC:
+
+<<code language="ocaml"|
+  let%rpc r = ...
```

Comment:
these are still not valid rpc syntax.

an rpc must have at least a parameter. Am I wrong ?